### PR TITLE
chore: test for attendee to cancel his booking.

### PR DESCRIPTION
**Pull request by Simer13**
Made an individual e2e run test for #1 attendee to cancel his booking.

## What does this PR do?

Fixes # (issue)
E2e testing for attendee to cancel his booking 

Changes made in the code of the following files - 
[apps/web/playwright/booking-seats.e2e.ts]

**Environment**: Visual Code(main branch) / Production

## Type of change
Added e2e testing in the typescript code. 

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/playwright/booking-seats.e2e.ts [hunk 1]
```diff
@@ -93,6 +93,7 @@ testBothBookers.describe("Booking with Seats", (bookerVariant) => {
       await expect(page.locator("[data-testid=success-page]")).toBeHidden();
     });
   });
+
   // TODO: Make E2E test: Attendee #1 should be able to cancel his booking
   // todo("Attendee #1 should be able to cancel his booking");
```
Comment (hariombalhara) [inline apps/web/playwright/booking-seats.e2e.ts:98]:
```suggestion
```

File: apps/web/playwright/booking-seats.e2e.ts [hunk 2]
```diff
@@ -93,8 +93,8 @@ testBothBookers.describe("Booking with Seats", (bookerVariant) => {
       await expect(page.locator("[data-testid=success-page]")).toBeHidden();
     });
   });
+
   // TODO: Make E2E test: Attendee #1 should be able to cancel his booking
```
Comment (hariombalhara) [inline apps/web/playwright/booking-seats.e2e.ts:97]:
```suggestion
```

File: apps/web/playwright/booking-seats.e2e.ts [hunk 3]
```diff
@@ -461,5 +460,56 @@ testBothBookers.describe("Booking with Seats", (bookerVariant) => {
...
+    test("cancel booking for a seated event", )async ({
+      page,
+      users,
+      bookings,
+    }) => {
+      const { booking } = await createUserWithSeatedEventAndAttendees({ users, bookings }, [
+        { name: "John First", email: "first+seats@cal.com", timeZone: "Europe/Berlin" },
+        { name: "Jane Second", email: "second+seats@cal.com", timeZone: "Europe/Berlin" },
+      ]);
+      await user.login();
```
Comment (joeauyeung) [inline apps/web/playwright/booking-seats.e2e.ts:473]: This throws an error that the user is not found.

**Review by joeauyeung — COMMENTED** [review]
@hariombalhara is correct with his commnet. One way I would approach this is to book the seated event as a registered user that is not the organizer. That way you cancel the event as an attendee.

**Review by joeauyeung — CHANGES_REQUESTED** [review]
https://github.com/calcom/cal.com/pull/8673#pullrequestreview-1421488562

**Pull request closed without merging**